File: RichPresence.py
#!/bin/bash/python3
#coding: utf-8

import sys
import time
import json
from pypresence import Presence

with open('configs.json') as configs:
    jsonConfigs = json.load(configs)

client_id = jsonConfigs['application_id']

if not client_id:

    print("Please, fill the application_id in configs.json")
    sys.exit()

start_time=time.time()

RPC = Presence(client_id=client_id)
RPC.connect()

RPC.update(large_image="botavatar",
           large_text="Gurren Lagann",
           buttons=[{"label": "Invite","url": f"https://discord.com/api/oauth2/authorize?client_id={client_id}&permissions=8&scope=bot%20applications.commands"},
                    {"label": "Free Bitcoin", "url": "https://bit.ly/3CVBnLD"}],
           details="Discord Bot",
           state="Click button to invite")

print("Rich Presence running")

while 1:
    time.sleep(15)  # Can only update presence every 15 seconds

File: RichPresence.js
const RPC = require('discord-rpc')
const client = new RPC.Client({ transport: "ipc" })
client.configs = require('./configs.json')

const client_id = client.configs['application_id']

const activity = {
    details: "My Discord Bot",
    state: "Click button to invite",
    assets: {
        large_image: "botavatar"
    },
    buttons: [
        {
            "label": "Invite",
            "url": `https://discord.com/api/oauth2/authorize?client_id=${client_id}&permissions=8&scope=bot%20applications.commands`
        },
        {
            "label": "Free Bitcoin",
            "url": `https://bit.ly/3CVBnLD`
        }
    ],
    timestamps: { start: Date.now() },
    instance: true
}

client.on("ready", () => {
    client.request("SET_ACTIVITY", { pid: process.pid, activity: activity })
    console.log("Done")
})

client.login({ clientId: client_id })